src/Rules/Comparison/ImpossibleCheckTypeFunctionCallRule.php

```diff
@@ -35,9 +34,6 @@ public function processNode(Node $node, Scope $scope): array
 		}
 
 		$functionName = (string) $node->name;
-		if (strtolower($functionName) === 'is_a') {
```

Comment:
because I dropped this early-return, I had to compensate a few false positives in the `IsAFunctionTypeSpecifyingExtension` - added a few early returns.